docs/source/starter/new-project.rst

```diff
@@ -12,35 +12,76 @@
 
 .. _new_project:
 
-####################
-Lightning in 2 steps
-####################
+###########
+Quick Start
+###########
 
-**In this guide we'll show you how to organize your PyTorch code into Lightning in 2 steps.**
+*****************
+Pytorch Lightning
+*****************
 
-Organizing your code with PyTorch Lightning makes your code:
+What is Pytorch Lightning
+=========================
 
-* Keep all the flexibility (this is all pure PyTorch), but removes a ton of boilerplate
-* More readable by decoupling the research code from the engineering
-* Easier to reproduce
-* Less error-prone by automating most of the training loop and tricky engineering
-* Scalable to any hardware without changing your model
+PyTorch has all you need to train your models. But there's a lot more to deep learning more than just attaching some layers together. It provides you the APIs required to build models, datasets etc.
+but when it comes to actual training, there's a lot of boiler-plate code involved that you need to write by youself and if you need to scale your training/inferencing on multiple devices/machines there's another
+set of integrations you might need to do by yourself. So here PyTorch Lightning comes into picture. All you need is some restructuring of your existing code and set certain flags and voila, you are done.
+Now you can train your models on different accelerators like GPU/TPU/IPU, do distributed training accross multiple machines/nodes with no code change use state-of-the-art distributed training mechanisms
+and a lot more. Now all you need is to take care of your research code and leave the engineering to us.
 
-----------
 
-Here's a 3 minute conversion guide for PyTorch projects:
+Why Pytorch Lightning
+=====================
 
-.. raw:: html
+Well, if you need one place that can help you with everything you need to train deep learning models, you are at the right place. The team makes sure that all the latest techniques are already integrated and maintained
+periodically to make sure you don't need to make any code changes.
+
+We test the framework across multiple Python/PyTorch versions to ensure its reach. It's not just Keras for PyTorch but much more. Follow along with the guide below to find out why.
```

Comment:
The Keras comparison is out of place. What does it mean to be "more than Keras" anyway?

Reply:
just wanted to specify that it's not just a wrapper around pytorch like keras where you plugin your model and do a `.fit`, but you have control over the loops and what happens inside each step.